sdcm/cluster.py

```diff
@@ -2548,60 +2548,27 @@ def get_console_screenshot(self):
         self.log.warning('Method get_console_screenshot is not implemented for %s' % self.__class__.__name__)
         return b''
 
-    def _resharding_status(self, status):
+    # Default value of murmur3_partitioner_ignore_msb_bits parameter is 12
+    def restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int = 12) -> None:
         """
-        Check is there's Reshard listed in the "nodetool compactionstats" output
-        status : expected values: "start" or "finish"
+        Resharding is started during Scylla startup and Scylla initialization has to wait until resharding done
         """
-        patt = re.compile('RESHARD|RESHAP')
-        result = self.run_nodetool("compactionstats", publish_event=False)
-        found = patt.search(result.stdout)
-        # wait_for_status=='finish': If 'RESHARD' is not found in the compactionstats output, return True -
-        # means resharding was finished
-        # wait_for_status=='start: If 'RESHARD' is found in the compactionstats output, return True -
-        # means resharding was started
-
-        return bool(found) if status == 'start' else not bool(found)
-
-    # Default value of murmur3_partitioner_ignore_msb_bits parameter is 12
-    def _restart_node_with_resharding(self, murmur3_partitioner_ignore_msb_bits: int = 12):
         self.stop_scylla()
         # Change murmur3_partitioner_ignore_msb_bits parameter to cause resharding.
         with self.remote_scylla_yaml() as scylla_yml:
             scylla_yml.murmur3_partitioner_ignore_msb_bits = murmur3_partitioner_ignore_msb_bits
-        search_reshard = self.follow_system_log(patterns=['Reshard', 'Reshap'])
+
+        search_reshard = self.follow_system_log(patterns=[DB_LOG_PATTERN_RESHARDING_START])
```

Comment:
Are you sure about this pattern?

DB_LOG_PATTERN_RESHARDING_START = "(?i)database - Resharding"

From what I saw we have "Resharding - for {name_of_table}"
and when it's done "Resharded XGB for {name_of_table}

Reply:
We have different log output from resharding. It's less important here. The fact that we have "Resharding" strings means that resharding is running